src/Compiler/tools/Microsoft.AspNetCore.Razor.SourceGenerator.Tooling.Internal/Microsoft.AspNetCore.Razor.SourceGenerator.Tooling.Internal.csproj

```diff
@@ -24,11 +24,10 @@
     <Compile Include="..\..\Microsoft.AspNetCore.Mvc.Razor.Extensions\src\**\*.cs" LinkBase="RazorExtensions" />
     <EmbeddedResource Include="..\..\Microsoft.AspNetCore.Mvc.Razor.Extensions\src\RazorExtensionsResources.resx"
       ClassName="Microsoft.AspNetCore.Mvc.Razor.Extensions.RazorExtensionsResources" />
-
-    <Compile Include="$(SharedSourceRoot)\Microsoft.AspNetCore.Razor.Utilities.Shared\LanguageSupport\NullableAttributes.cs" LinkBase="LanguageSupport" />
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="$(SharedSourceRoot)\Microsoft.AspNetCore.Razor.Utilities.Shared\Microsoft.AspNetCore.Razor.Utilities.Shared.csproj" />
```

Comment:
Is the right change? There's so much care taken to link the source files from other projects. Why aren't the other projects just references?

Reply:
I frankly don't know why the others aren't just project references either :). @chsienki, do you have opinions on this before I hit merge?

Reply:
Hmm, good catch. Honestly I'm not even sure anything is using this package anymore, but it's still being consumed from the SDK. It was all source linked because it was shipped as a single assembly, and the tooling in the SDK only picked up the one assembly from the package. 

I'll open an issue to track removing this project altogether, but until then lets source link this too, to make sure we don't break anything downstream that is still relying on it.

Reply:
Alright. @DustinCampbell, I'll wait till your change is merged, then I'll merge and update this PR.

Reply:
Looks like I'll need a thumbs up on that PR before I can merge.